src/shadowbox/server/api.yml

```
@@ -203,6 +203,13 @@ paths:
       description: Creates a new access key with a specific identifer
       tags:
         - Access Key
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: The id to get the access key
```

Could you fix the description?

```suggestion
          description: The id for which to create an access key
```